index.js

```diff
@@ -0,0 +1,22 @@
+var recipes = new Object();
+
+function updateObjectWithKeyAndValue(object, key, value){
+  return Object.assign({},object,{[key]: value})
+}
+
+function destructivelyUpdateObjectWithKeyAndValue(object, key, value){
+ object[key] = value
+return object
+}
+
+function deleteFromObjectByKey(object, key){
+  var newObj =  Object.assign({},object)
+  delete newObj[key]
+  return newObj
+
```

Comment:
Friendly suggestion: remove line 16 to remove the empty space :)

Unless it is on purpose?

Reply:
const driver = Object();

function updateDriverWithKeyAndValue(obj, key, value) {
    let Object = {...obj };
    Object[key] = value;

    return Object;
}